roles/neutronstar/templates/neutronstar_site.config

```diff
@@ -0,0 +1,13 @@
+process {
+    container = '{{ neutronstar_container_path }}/{{ neutronstar_image }}'
+    withName:multiqc {
+        container = null
+        executor = 'local'
+    }
+}
+params {
+    project = "{{ item.project_id }}"
+    multiqc_config = "{{ ngi_pipeline_conf }}/multiqc_{{ item.site }}_config.yml"
+    clusterOptions = "-A {{ ngi_pipeline_sthlm_delivery }} -p node"
```

Comment:
this should be `{{ item.project_id }}`, right?

Reply:
Correct, good spot 👍 

Looks like this error was there before (see [above](https://github.com/NationalGenomicsInfrastructure/irma-provision/pull/300/files#diff-7beecd24084a8d96a56f894e0e69a217L36)) and I just moved it over. Kind of surprised that it ever worked.. Maybe it didn't?